benchmarks/xgraph/subgraphx_tg_run.py

```diff
@@ -112,6 +112,7 @@ def pipeline(config: DictConfig):
     # DONE: only use tgat processed data
     events, edge_feats, node_feats = load_tg_dataset(config.datasets.dataset_name)
     target_event_idxs = load_explain_idx(config.datasets.explain_idx_filepath, start=0)
+    target_event_idxs = target_event_idxs[:10]
```

Comment:
Is this intended?

Reply:
Nope, it somehow sneaked in while testing. I debated a little about a config option that limits the number of events to explain though.